src/components/ProblemSolutionTable/constants.ts

```diff
@@ -6,13 +6,15 @@ export type HeaderCell = {
   id: string;
   label: string;
   sortable: boolean;
+  width?: number;
```

Comment:
for fixed column size. Only column that varies is the problem title, others are usually small